Doc/reference/datamodel.rst

```diff
@@ -453,7 +453,7 @@ Sets
 
    These represent a mutable set. They are created by the built-in :func:`set`
    constructor and can be modified afterwards by several methods, such as
-   :meth:`~set.add`.
+   :meth:`~add <frozenset.add>`.
```

Comment:
```suggestion
   :meth:`add <frozenset.add>`.
```

And is there another re-org PR in progress to allow us to link properly to `set.add`?